config_defaults_inc.php

```diff
@@ -2591,6 +2591,24 @@
 	'view_state',
 );
 
+/**
+ * An array of access levels or a threshold to identify who should be presented the extended  report form.
+ * Setting it to an array, one can cherry-pick the access levels that should get the extended report screen.
+ * sample: $g_extended_reporter_levels = array( DEVELOPER, ADMINISTRATOR );
+ * Now only the Developers & Administrators will get the extended report form, all others will get the simplified report form.
+ *
+ * @global array $g_extended_report_levels
+ */
+$g_extended_reporter_levels = REPORTER;
```

Comment:
Must be 
```
$g_extended_report_levels = REPORTER;
```

Reply:
Have corrected the typo and squashed the 2 commits into one(1)